### DISCUSSION TITLE
Support for more SASS utilities

### DISCUSSION THREAD

**Discussion by 0xMurage** [vmware-clarity/ng-clarity#244] [Q&A] [answered]
Are there intentions to add more SCSS utilities to @clr/ui ?
I have worked with Bootstrap and I find clr/ui short of my expectations . e.g. for display utilities, Clarity only provides `clr-display-block`, `clr-display-inline-block` and `clr-display-inline`  while Bootstrap provides `d-inline` `d-inline-block` `d-block grid` `d-table` `d-table-row` `d-table-cell` `d-flex` `d-inline-flex` and `d-none`.

I also don't see a reason why a dev can't have a better way to customize what get included in their final build. There is too much separation of responsibilities which is too inconvenient without docs on customization;
 for comparison, here is [boostrap entry file ](https://github.com/twbs/bootstrap/blob/main/scss/bootstrap.scss)https://github.com/twbs/bootstrap/blob/main/scss/bootstrap.scss vs [Clarity UI](https://github.com/vmware-clarity/ng-clarity/blob/main/projects/angular/src/utils/_components.clarity.scss) https://github.com/vmware-clarity/ng-clarity/blob/main/projects/angular/src/main.scss

**Comment by ashleyryan** [comment, marked as answer]
We don't have plans to add more SCSS utilities to clr/ui. We're in the process of shimming clr/ui to align better with the CSS Variables and layout utilities in in Clarity Core. 

Checkout our layout documentation for Core here: https://storybook.core.clarity.design/?path=/story/layout-get-started--page
